### PR TITLE
U02: Add endpoint for updating player name

### PR DESCRIPTION
**Pull request by Twhining**

### CODE REVIEW

File: src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java [hunk 1]
```diff
@@ -57,6 +57,12 @@ public DataDTO getDataByPlayerId(@RequestBody String playerId) {
         return createDataDTO(session, players, tiles);
     }
 
+    @PutMapping("/name")
```
Comment (paul891112) [inline src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java:60]: Would be nice to have HttpStatus.OK (200) or HttpStatus.NOCONTENT(204) as response.

I dont need it in front-end, so keeping it like this is fine too

**Review by paul891112 — APPROVED** [review]

File: src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java (hunk 1 above)
Comment (Twhining) [reply src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java:60]: the default response is HttpStatus.OK

Comment (paul891112) [reply src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java:60]: Perfect thanks!

**Review by C0DECYCLE — CHANGES_REQUESTED** [review]
@Twhining, please wait till the backend refactor is merged and then check if this pr needs adapting. I will look at it afterwards. Thanks a lot!

**Pull request closed without merging**